src/frontend/Sider/Behandling/Vilkårvurdering/EndreDelvilkår.tsx

```diff
@@ -134,40 +137,58 @@ const EndreVurderingComponent: FC<{
 
     return (
         <form onSubmit={onSubmit}>
-            <Container>
+            <VStack gap="4">
                 {delvikårsett.map((delvikår, delvikårIndex) => {
                     return delvikår.vurderinger.map((svar, indeks) => {
-                        const regel = regler[svar.regelId];
+                        const gjeldendeRegel = regler[svar.regelId];
+                        const erUndervilkår = indeks !== 0;
                         return (
-                            <DelvilkårContainer $erUndervilkår={indeks !== 0} key={regel.regelId}>
-                                <VurderDelvilkår
-                                    vurdering={svar}
-                                    regel={regel}
-                                    settVurdering={(nyVurdering) =>
-                                        oppdaterSvar(
-                                            delvikår.vurderinger,
-                                            delvikårIndex,
-                                            nyVurdering
-                                        )
-                                    }
-                                />
-                                <Begrunnelse
-                                    onChange={(begrunnelse) =>
-                                        oppdaterBegrunnelse(delvikår.vurderinger, delvikårIndex, {
-                                            ...svar,
-                                            begrunnelse,
-                                        })
-                                    }
-                                    svar={svar}
-                                    regel={regel}
-                                />
-                            </DelvilkårContainer>
+                            <>
+                                {delvikårIndex !== 0 && !erUndervilkår && <Skillelinje />}
```

Comment:
Lurer på om du kunne fjernet `delvilkårIndex !== 0` flyttet <Skillelinje/> under `</DelvilkårContainer>` og fjernet den fra å ligge med lagreknappen

Reply:
Prøvde - ble litt for mange skillelinjer 🤷‍♂️